apps/charterafrica/src/payload/fields/countries.js

```diff
@@ -0,0 +1,1972 @@
+const countries = [
```

Comment:
There is also countries (African) used in openAFRICA. Shouldn't we combine these 2 data files?